frontend/components/card/CardDatePicker.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+    <form v-if="$colorMode.preference == 'light'" class="flex flex-col w-full space-y-3 px-5 py-4 card-style"
+      
+      
+    >
+    <h1 class="text-2xl" ><strong>Date *</strong></h1>
+
+<div class="flex justify-between">
+  <div class="flex flex-row space-x-2">
+    <input id="All Day" type="checkbox"  value="All Day" />
+    <label for="All Day">All Day</label>
+  </div> 
+
+  <div class="flex flex-row space-x-2 margin-left-20">
+    <input id="Multiple Days" type="checkbox"  value="Multiple Days" />
+    <label for="Multiple Days">Multiple Days</label>
+  </div>
+</div>
+
+<div class="flex flex-row">
+  <DatePicker
+    v-model="date1" mode="Time" is24hr hide-time-header
+    class="date-picker custom-gray-datepicker"
+    borderless
+    />
+  <DatePicker
+    v-model="date2" mode="Time" is24hr hide-time-header
+    class="date-picker custom-gray-datepicker"
+    borderless
+    />
+</div>
+
+<DatePicker
+  v-model="dateArray"  
+  :first-day-of-week="2"
+  trim-weeks
+  expanded
+  :attributes="attributes"
+  transparent
+  borderless
+  ></DatePicker>
+    </form>
+    
+    <form v-else class="flex flex-col w-full space-y-3 px-5 py-4 card-style">
+      <h1 class="text-2xl" ><strong>Date *</strong></h1>
+
+      <div class="flex-container">
+        <div class="flex flex-row space-x-2">
+          <input id="All Day" type="checkbox"  value="All Day" />
+          <label for="All Day">All Day</label>
+        </div> 
+
+        <div class="flex flex-row space-x-2 ml-right">
+          <input id="Multiple Days" type="checkbox"  value="Multiple Days" />
+          <label for="Multiple Days">Multiple Days</label>
+        </div>
+      </div>
+
+      <div class="flex flex-row">
+        <DatePicker
+          v-model="date1" mode="Time" is24hr hide-time-header
+          class="date-picker custom-gray-datepicker"
+          borderless
+          />
+        <DatePicker
+          v-model="date2" mode="Time" is24hr hide-time-header
+          class="date-picker custom-gray-datepicker"
+          borderless
+          />
+      </div>
+
+      <DatePicker
+        v-model="dateArray"
+        :first-day-of-week="2"
+        trim-weeks
+        expanded
+        :attributes="attributes"
+        transparent
+        borderless
+        ></DatePicker>
+    </form>
+  </template>
+  
+  <script setup>
```

Comment:
Also no stress as TS wasn't causing any errors anyway, but for the `<script>` block we need it defined as `<script setup lang="ts">` to turn on TS :)